src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -63,10 +71,11 @@ class PropertyAccessor implements PropertyAccessorInterface
      * Should not be used by application code. Use
      * {@link PropertyAccess::createPropertyAccessor()} instead.
      */
-    public function __construct($magicCall = false, $throwExceptionOnInvalidIndex = false)
+    public function __construct($magicCall = false, $throwExceptionOnInvalidIndex = false, Cache $cache = null)
```

Comment:
I think it is ok. Doctrine cache dependency is currently used in Symfony:

- https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Serializer/Mapping/Factory/ClassMetadataFactory.php#L14
- https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Validator/ValidatorBuilder.php#L17